new-character.lic

```diff
@@ -90,7 +90,13 @@ class NewCharacter
   def check_theurgy
     echo '***CHECKING THEURGY***'
     return unless @settings.crossing_training.include?('Theurgy')
-    return unless DRStats.cleric?
+
+    if DRStats.paladin?
```

Comment:
Will the `return unless` on line 92 prevent this from running? Or do Paladins put Theurgy in their training list?

It feels like this might go better in the 'manual steps' section.

Reply:
Good call, done. I looked for badge and din't know about manual_changes.